### PR TITLE
XW-1317 | run PR checks simultaneously on different ports

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1317

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: tasks/jenkins/mercury-pr-checks.sh [hunk 1]
```diff
@@ -120,7 +120,7 @@ fi
 
 ### Front tests - running
 updateGit "Front tests" pending running
-npm run test-front 2>&1 | tee jenkins/front-tests.log || { error1=true && failJob=true; }
+{ gulp test-common && gulp test-auth && cd front/main && ember test --path ../../www/front/main -tp $((7357 + $EXECUTOR_NUMBER)); } 2>&1 | tee jenkins/front-tests.log || { error1=true && failJob=true; }
```
Comment (rogatty) [inline tasks/jenkins/mercury-pr-checks.sh]: Can we use `npm run test-front` and pass a param? This is additional place to update build process and surely someone will forget about it.

**Comment by hakubo** [comment]
There is also .ember-cli.json file that we could use; http://ember-cli.com/#runtime-configuration

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "start": "nodemon --delay 2",
     "start-on-change": "nodemon --delay 2 --on-change-only",
     "test": "npm run build-dev && npm run test-front && npm run test-server",
-    "test-front": "gulp test-common && gulp test-auth && cd front/main && ember test --path ../../www/front/main",
+    "test-front": "gulp test-common && gulp test-auth && cd front/main && ember test --path ../../www/front/main -tp $((7357 + $TEST_PORT))",
```
Comment (hakubo) [inline package.json]: cool, that looks good
althogh when I run this in zsh I get this
zsh: bad math expression: operand expected at end of string

**Comment by vforge** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
Tested locally, on devbox and on Jenkins

**Pull request merged**